CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+# Contributing
+
+First of all, read how to deploy a local copy of Pan Docs in the [README](https://github.com/gbdev/pandocs#contributing).
+
+## Git
+
+- Commit messages must be meaningful, avoid "Update XXX.md". Please describe what you changed, how, and **why**.
+- If it is desirable to squash a PR, but not into a single commit, keep in mind that most of the time, maintainers have push access to branches being PR'd ("Allow edits and access to secrets by maintainers" checkbox in the right sidebar).
```

Comment:
Maybe we can reassure the potential contributor with "Don't worry if the commit history and messages aren't perfect, maintainers can clean it up within reason"?